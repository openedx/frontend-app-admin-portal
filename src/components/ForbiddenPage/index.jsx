import React from 'react';
import { Helmet } from 'react-helmet';
import { MailtoLink } from '@edx/paragon';

const ForbiddenPage = () => (
  <main role="main">
    <div className="container-fluid mt-3">
      <Helmet>
        <title>Access Denied</title>
      </Helmet>
      <div className="text-center py-5">
        <h1>403</h1>
        <p className="lead">You do not have access to this page.</p>
        <p>
          For assistance, please contact the edX Customer Success team at
          {' '}
          <MailtoLink to="customersuccess@edx.org">customersuccess@edx.org</MailtoLink>.
        </p>
      </div>
    </div>
  </main>
);

export default ForbiddenPage;
