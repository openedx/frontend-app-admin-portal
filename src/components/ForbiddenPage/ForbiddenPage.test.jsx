import React from 'react';
import renderer from 'react-test-renderer';

import ForbiddenPage from './index';

describe('<ForbiddenPage />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <ForbiddenPage />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
