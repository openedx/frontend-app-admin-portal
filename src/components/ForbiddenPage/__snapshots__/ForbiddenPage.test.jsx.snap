// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForbiddenPage /> renders correctly 1`] = `
<main
  role="main"
>
  <div
    className="container-fluid mt-3"
  >
    <div
      className="text-center py-5"
    >
      <h1>
        403
      </h1>
      <p
        className="lead"
      >
        You do not have access to this page.
      </p>
      <p>
        For assistance, please contact the edX Customer Success team at
         
        <a
          className="pgn__hyperlink default-link standalone-link pgn__mailtolink"
          href="mailto:customersuccess@edx.org"
          onClick={null}
          target="_self"
        >
          customersuccess@edx.org
        </a>
        .
      </p>
    </div>
  </div>
</main>
`;
